web/tendrl/user_password_edit_dialog.rst

```diff
@@ -0,0 +1,67 @@
+Change user password at user "Edit" page
+*****************************************
+
+:authors: 
+          - mkudlej@redhat.com
+
+Description
+===========
+
+Change user password at user "Edit" page for these user types:
+ 1. *Admin* user
+ 2. *Power User* user
+ 3. *Read-only* user
+
+Design describing "Edit" page `https://redhat.invisionapp.com/share/KNB25OEQT#/screens/226063805`
```

Comment:
This syntax would make the link shown in cursive, to create a link for that url without specifying a link text, you can just use the url without any special syntax:

```
Design describing "Edit" page https://redhat.invisionapp.com/share/KNB25OEQT#/screens/226063805
```